Remotion/ObjectBinding/Web.ClientScript/Scripts/BocList.ts

```diff
@@ -123,6 +123,14 @@ class BocList
       selectedRows.SelectAllSelectorControls = document.querySelectorAll<HTMLInputElement>('input[name="' + selectAllSelectorControlName + '"]')!;
       selectedRows.OnSelectionChanged = onSelectionChangedHandler;
 
+      var visualizer = document.querySelectorAll('input[name="' + selectAllSelectorControlName + '"] + span'/*.checkbox-visualizer*/)!;
+      visualizer.forEach(checkbox => {
+          checkbox.addEventListener('click', function (evt) {
+              evt.stopPropagation();
+                  selectedRows.SelectAllSelectorControls![0]!.click();
```

Comment:
is the indent incorrect? also please check the braces, we use a different layout than here in the diff.